src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -35,6 +35,24 @@ service Admin {
   rpc ScaleCluster(ScaleClusterRequest) returns (google.rpc.Status) {
     option (google.api.http) = { get: "/v1test/admin:scaleCluster" };
   }
+  
+  rpc ReindexCas(ReindexCasRequest) returns (ReindexCasRequestResults) {
+    option (google.api.http) = { get: "/v1test/admin:reindexCas" };
+  }
+}
+
+message ReindexCasRequest {
+  string instance_name = 1;
+  
+  string host_id = 2;
```

Comment:
What are we using as host ID? A full worker name in the IP:PORT format? Or just IP address? Maybe either a host_name or worker_name would be better here. Also can you add a comment here on what the expected format is.

Reply:
Yes, i keep wondering about the same thing when going through the code, i think it's `ip:port`, like `localhost:8981`.

Reply:
I was also unsure about the name.  Other "request types" use `host_id` so I followed that pattern, ex:
```
message TerminateHostRequest {
  string instance_name = 1;
  string host_id = 2;
}
```

In this case, `host_id` is the `ip:port` because it correlates to the key format that's in `smembers  ContentAddressableStorage:*`.  I'll document that it should be that format. But also willing to change the name if it can improve clarity.